### DISCUSSION TITLE
Adding features to the site/app

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#15] [Ideas]
Am thiniking about making a 200px-250px left nav panel that may be a offcanvas drawer that has a bunch of features within it.
Here are my ideas, with their routing:

Group routes to <url>/groups/feed/ or /g/ with a group id as groups/{group_id} may also hav its own chat with /groups/chat/ or groups/messages/ with some other features as well.
Chat Messages route to <url>/messages/  (this is linked to the messages icon in the top bar)
Notifications route to <url>/notifications/ (this is linked to the bell icon)
Feed routes to <url>/feed/ (this is what you would see in the main / center panel)
Messages route to <url>/inbox/  (this would be coming from ?)
Friends (a list of your contacts/friends) routes to /friends/
Events routes to <url>/events/ (this ideally has panels for displaying a calendar of events)
Search routes to <url>/search/ (this is results from the search bar)

This project uses Material  UI // mui v5

**Comment by gbowne1** [comment]
@balajik

**Comment by PushpitSB** [comment]
how about one more features calendar according to priority of the events like how urgent they are.

**Reply by gbowne1** [reply]
@PushpitSB that is a great idea.
